skills/unraid/scripts/dashboard.sh

```
@@ -0,0 +1,229 @@
+    echo "- **Memory:** ${TOTAL_MEM_GB}GB - **${MEM_USED_PCT}% used**" >> "$OUTPUT_FILE"
+    echo "" >> "$OUTPUT_FILE"
+
+    # Array capacity
+    ARRAY_TOTAL=$(echo "$RESPONSE" | jq -r '.data.array.capacity.kilobytes.total')
+    ARRAY_USED=$(echo "$RESPONSE" | jq -r '.data.array.capacity.kilobytes.used')
+    
+    if [ "$ARRAY_TOTAL" != "null" ] && [ "$ARRAY_TOTAL" -gt 0 ]; then
+        ARRAY_TOTAL_GB=$((ARRAY_TOTAL / 1024 / 1024))
+        ARRAY_USED_GB=$((ARRAY_USED / 1024 / 1024))
+        ARRAY_USED_PCT=$((ARRAY_USED * 100 / ARRAY_TOTAL))
+        echo "### Storage" >> "$OUTPUT_FILE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)